gradle/modules.properties

```diff
@@ -1,8 +1,3 @@
-# Add an entry here, if you add a dependency that has a 'module-info.class', but can't be found.
-# If this is needed, there is no entry yet in one of these:
-#   https://github.com/gradlex-org/java-module-dependencies/blob/main/src/main/resources/org/gradlex/javamodule/dependencies/unique_modules.properties
-#   https://github.com/gradlex-org/java-module-dependencies/blob/main/src/main/resources/org/gradlex/javamodule/dependencies/modules.properties
-# Consider contributing the missing entry back to the plugin by creating a PR for the 'modules.properties' linked above.
```

Comment:
Is this comment no longer correct?

Reply:
No it is still correct. I felt it was too noisy and maybe not very relevant when I went over everything again. But maybe it is good to keep. In any case, it is not important for this PR to change anything here. I will revert this.